src/javascripts/components/forms/staffMemberRole.js

```diff
@@ -0,0 +1,68 @@
+import { getDinos } from '../../helpers/data/dinoData';
+import { getRides } from '../../helpers/data/ridesData';
+import { getVendors } from '../../helpers/data/vendorData';
+
+const staffMemberRole = (role) => {
+  document.querySelector('#specific-activity').innerHTML = '';
+  // let domString = `
```

Comment:
Is this commented out code needed for something moving forward?

Reply:
resolved, thank you.